packages/webview_flutter/tizen/src/webview.cc

```diff
@@ -154,19 +155,27 @@ double ExtractDoubleFromMap(const flutter::EncodableValue& arguments,
 }
 
 WebView::WebView(flutter::PluginRegistrar* registrar, int viewId,
-                 FlutterTextureRegistrar* texture_registrar, double width,
+                 flutter::TextureRegistrar* texture_registrar, double width,
                  double height, flutter::EncodableMap& params)
     : PlatformView(registrar, viewId),
       texture_registrar_(texture_registrar),
       webview_instance_(nullptr),
       width_(width),
       height_(height),
-      tbm_surface_(nullptr),
+      candidate_surface_(nullptr),
+      rendered_surface_(nullptr),
       is_mouse_lbutton_down_(false),
       has_navigation_delegate_(false),
       has_progress_tracking_(false),
       context_(nullptr) {
-  SetTextureId(FlutterRegisterExternalTexture(texture_registrar_));
+  flutter::TextureVariant* textureVariant_ =
+      new flutter::TextureVariant(flutter::GpuBufferTexture(
+          [this](size_t width,
+                 size_t height) -> const FlutterDesktopGpuBuffer* {
+            return this->CopyGpuBuffer(width, height);
+          },
+          [this](void* buffer) -> void { this->Destruction(buffer); }));
```

Comment:
I think it is dangerous that capturing ```this``` pointer. because ```plugin instance``` will destroy before shutdown engine, and some operation related this can be occurred in raster task thread.

Reply:
I thought that part was solved by @xiaowei-guan 's  last work, but there must be a left something... I'll check what mentioned above.

Reply:
I took a look a while.. but I can't find that situation you were mentioned above...